pkg/services/cosmosdb/common_types.go

```diff
@@ -1,46 +1,14 @@
-// +build experimental
-
 package cosmosdb
 
-import (
-	"fmt"
-	"net"
-
-	"github.com/Azure/open-service-broker-azure/pkg/ptr"
-	"github.com/Azure/open-service-broker-azure/pkg/service"
-)
-
-type provisioningParameters struct {
-	IPFilterRules     *ipFilterRule      `json:"ipFilters"`
-	ConsistencyPolicy *consistencyPolicy `json:"consistencyPolicy,omitempty"`
-}
-
-type ipFilterRule struct {
-	Filters     []string `json:"allowedIPRanges"`
-	AllowAzure  string   `json:"allowAccessFromAzure,omitempty"`
-	AllowPortal string   `json:"allowAccessFromPortal,omitempty"`
-}
-
-type boundedStaleness struct {
-	MaxStaleness *int `json:"maxStalenessPrefix,omitempty"`
-	MaxInternal  *int `json:"maxIntervalInSeconds,omitempty"`
-}
-
-type consistencyPolicy struct {
-	DefaultConsistency string            `json:"defaultConsistencyLevel,omitempty"`
-	BoundedStaleness   *boundedStaleness `json:"boundedStaleness,omitempty"`
-}
+import "github.com/Azure/open-service-broker-azure/pkg/service"
 
 type cosmosdbInstanceDetails struct {
 	ARMDeploymentName        string `json:"armDeployment"`
 	DatabaseAccountName      string `json:"name"`
 	FullyQualifiedDomainName string `json:"fullyQualifiedDomainName"`
 	IPFilters                string `json:"ipFilters"`
-}
-
-type cosmosdbSecureInstanceDetails struct {
-	ConnectionString string `json:"connectionString"`
-	PrimaryKey       string `json:"primaryKey"`
+	ConnectionString         string `json:"connectionString"`
+	PrimaryKey               string `json:"primaryKey"`
```

Comment:
@jeremyrickard shouldn't these two fields be of type `service.SecureString`?

Reply:
Absolutely should, good catch! Updating.